hw/ip_templates/rstmgr/data/rstmgr.hjson.tpl

```diff
@@ -179,9 +182,11 @@
     { name: "RSTMGR.CPU_INFO.ENABLE",
       desc: "Enable capture of cpu crash dump information."
     }
+% if with_alert_handler:
```

Comment:
I think this is orthogonal to the lack of alert_dump. It is possible there is a rstmgr that is not the ROT's (say it is from subsystem XYZ), and the alert dump is sent to the ROT's rstmgr. But there could be alert senders in XYZ, which means they would need to respond to alert sender pings, so the ROT's alert handler would need to know then those senders are unavailable, thus XYZ's rstmgr would need to send RESET_STATUS for those alert senders.